adapters/adman/usersync_test.go

```diff
@@ -0,0 +1,30 @@
+package adman
+
+import (
+	"testing"
+	"text/template"
+
+	"github.com/prebid/prebid-server/privacy"
+	"github.com/prebid/prebid-server/privacy/gdpr"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAdmanSyncer(t *testing.T) {
+	syncURL := "https://sync.admanmedia.com/pbs.gif?redir=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dadman%26uid%3D%5BUID%5D"
```

Comment:
Please include the GDPR and GDPRConsent macros in the test. It's usually a literal copy and paste from config.go with the host hardcoded (the latter you have here already).